docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -10,19 +10,52 @@ Chipyard is developed and tested on Linux-based systems.
 
 .. Warning:: Working under Windows is not recommended.
 
+Running on AWS EC2 with FireSim
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In CentOS-based platforms, we recommend installing the following dependencies:
+If you plan on using Chipyard alongside FireSim, you should refer to the `FireSim documentation `<https://docs.fires.im/>`__
+on "Initial Setup/Simulation".
+When you reach Section 2.3.2 on "Setting up the FireSim Repo"
+you can continue to :ref:`Setting up the Chipyard Repo` to continue to setup the repository with Chipyard as the top-level repository.
```

Comment:
Can you directly link to the correct starting section of the FireSim docs?

Reply:
That's the plan. I just need to figure out to point to either latest or stable depending on what the version of the CY docs is being built.